schemas/WebPanelResize.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "object",
+  "description": "Triggered when the user resizes the right panel.",
```

Comment:
I wonder if this is meant to mention the `right` panel given there's a `panel` property below which could extend this for more panel locations?

Reply:
Good spot